me.gladwell.eclipse.m2e.android/plugin.xml

```diff
@@ -1,51 +1,54 @@
-<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
I think so. I think a repo should have a consistent line-ending policy (which should be unix). And it is only six git commands, which can be copied from that article.